rcnn/BatchedNms.cu

```diff
@@ -0,0 +1,124 @@
+#include "BatchedNmsPlugin.h"
+#include "cuda_utils.h"
+
+#include <algorithm>
+#include <iostream>
+#include <stdexcept>
+#include <cstdint>
+#include <vector>
+#include <cmath>
+
+#include <cuda.h>
+#include <thrust/device_ptr.h>
+#include <thrust/sequence.h>
+#include <thrust/execution_policy.h>
+#include <thrust/gather.h>
+#include <thrust/system/cuda/detail/cub/device/device_radix_sort.cuh>
+
```

Comment:
what is `thrust`? do we need extra steps to install it?

Reply:
No, just need to install cuda. thrust is located in the cuda include directory

Reply:
OK, cool.